src/pages/gateway/batching.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Batching with API Mesh
+description: Learn how API Mesh uses batching. 
+kewords:
+  - API Mesh
+  - Extensibility
+  - GraphQL
+  - Integration
+  - REST
+  - Tools
+---
+
+# Batching with API Mesh for Adobe Developer App Builder
+
+Batching allows you to combine a group of requests into a single request, turning multiple queries into a single one. Compared to sending multiple queries simultaneously, batched requests result in better response times. They also avoid issues with rate-limiting.
+
+Consider a scenario where you are using the following mesh, where `<reviews_api>` is a third-party API that contains reviews for your products by SKU. Each review has a `review`, `customer_name`, and `rating` field.
+
+```json
+{
+    "meshConfig": {
+        "sources": [
+            {
+                "name": "Products",
+                "handler": {
+                    "graphql": {
+                        "endpoint": "https://venia.magento.com/graphql"
+                    }
+                }
+            },
+            {
+                "name": "Reviews API",
+                "handler": {
+                    "graphql": {
+                        "endpoint": "<Reviews_API_URL>",
+                        "useGETForQueries":true
+                    }
+                }
+            }
+        ]
+    }
+}
+```
+
+The following query causes multiple calls to the Reviews API:
+
+```graphql
+{
+  productsReviews(sku: ["VT12","VD03"]) {
+    sku
+    reviews {
+      review
+      customer_name
+      rating
+    }
+  }
+}
+```
+
+To make a single network request to each source, modify the mesh configuration to contain the `addtionalTypeDef` and `additionalResolver` described below:
```

Comment:
If we modify this statement to "To make a single network request to reviews source for the different product skus, modify the mesh configuration to contain the `additionalTypeDef` and `additionalResolver` described below:", It will be more helpful.